packages/tasit-account/src/ethers.test.js

```diff
@@ -0,0 +1,176 @@
+import { expect, assert } from "chai";
+import { ethers } from "ethers";
+
+const waitForEvent = async (eventName, expected) => {
+	return new Promise(function(resolve, reject) {
+		contract.on(eventName, function() {
+			const args = Array.prototype.slice.call(arguments);
+			const event = args.pop();
+			event.removeListener();
+			expect(args, `${event.event} event should have expected args`).to.deep.equal(
+				expected
+			);
+			resolve();
+		});
+	});
+};
+
+const ropstenProvider = ethers.getDefaultProvider("ropsten");
+const zero = ethers.utils.bigNumberify(0);
+
+let wallet, randomWallet, contract;
+
+const rawTx = {
+	nonce: 0,
+	gasLimit: 21000,
+	gasPrice: ethers.utils.bigNumberify("20000000000"),
+	to: "0x88a5C2d9919e46F883EB62F7b8Dd9d0CC45bc290",
+	value: ethers.utils.parseEther("0.0000001"),
+	data: "0x",
+	chainId: ethers.utils.getNetwork("ropsten").chainId,
+};
+
+describe("ethers.js - pretests", function() {
+	it("should create a wallet from priv key and provider", async function() {
+		const privateKey =
+			"0x18bfdd05c1e63e4a27081352e2910c164a4d34588f8d7eecfdfcb654742934c2";
+
+		wallet = new ethers.Wallet(privateKey, ropstenProvider);
+		expect(wallet.address).to.have.lengthOf(42);
+		expect(wallet.provider).to.be.not.undefined;
+	});
+
+	it("should create a random wallet ", async function() {
+		randomWallet = ethers.Wallet.createRandom();
+		expect(randomWallet.address).to.have.lengthOf(42);
+	});
+
+	it("should get balance of wallet", async function() {
+		randomWallet = randomWallet.connect(ropstenProvider);
+
+		const fundedWalletBalance = ethers.utils.bigNumberify(
+			await wallet.getBalance()
+		);
+		const emptyWalletBalance = ethers.utils.bigNumberify(
+			await randomWallet.getBalance()
+		);
+
+		expect(fundedWalletBalance).not.to.be.undefined;
+		assert(emptyWalletBalance.eq(zero));
+	});
+});
+
+// Note: We're intentionally not testing the `fromEncryptedJson` or `encrypt` functions
+// from `ethers.js` because we don't plan to expose that functionality in the Tasit SDK.
+// For a detailed explanation of why, see this GitHub issue:
+// https://github.com/tasitlabs/TasitSDK/issues/24#issuecomment-443576993
+describe("ethers.js - unit tests", function() {
+	before("wallet used to sign transactions should have funds", async () => {
+		const balance = ethers.utils.bigNumberify(await wallet.getBalance());
+
+		assert(balance.gt(zero), "wallet balance is zero");
+	});
+
+	it("should sign a raw transaction", async function() {
```

Comment:
@marcelomorgado I'll note that an ethers test of sending a raw transaction will eventually move to `tasit-act` or whatever we end up calling the reading and writing from the blockchain. That is if we decide to keep raw ethers tests in the repo.

Reply:
Ok